src/DateUtils.js

```diff
@@ -15,9 +15,36 @@ limitations under the License.
 */
 
 'use strict';
+import _t from 'counterpart';
 
-var days = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
-var months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+function getDaysArray() {
+	var days = [];
+	days.push(_t('Sun'));
```

Comment:
i'm still very suspicious that the array literal doesn't internationalise correctly.  Perhaps it's an autohoisting thing - does it work if you do `var days;` and then `days = [ _t('Mon'), ... ]`?